src/stream/src/executor/hash_agg.rs

```diff
@@ -491,28 +559,38 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
                             )?;
                         }
                         if let Some(prev_outputs) = prev_outputs {
-                            let old_row = agg_group.group_key().chain(prev_outputs);
+                            // FIXME: double deserialization here
+                            let group_key = K::deserialize(key, group_key_data_types).unwrap();
+                            let old_row = group_key.chain(prev_outputs);
```

Comment:
Why can't use `agg_group.group_key()` here?

Reply:
OK for now. Someone tells me we'll remove the group_key() later.